include/deal.II/dofs/dof_tools.h

```diff
@@ -2556,7 +2556,7 @@ namespace DoFTools
   unsigned int
   max_dofs_per_vertex (const hp::DoFHandler<dim,spacedim> &dh)
   {
-    return dh.get_fe().max_dofs_per_vertex ();
+    return dh.get_fe_collection().max_dofs_per_vertex ();
```

Comment:
I think the three functions `max_dofs_per_cell`, `max_dofs_per_face` and `max_dofs_per_vertex` do not need to be specialized for the two types of `DoFHandler` any more, so one can directly use `dh.get_fe_collection().max_dofs_per_xxx()` in all using sites, right?